Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCiscoMeraki.yaml

```diff
@@ -1,46 +1,580 @@
+        "18", "Invalid TCP",
+        "19", "Invalid TCP",
+        "20", "Invalid TCP",
+        "21", "Unknown",
+        "22", "Invalid TCP",
+        "23", "Invalid Tunnel",
+        "24", "Invalid Tunnel",
+        "32", "Unknown",
+        "33", "Invalid TCP",
+        "34", "Invalid TCP",
+        "35", "Invalid TCP",
+        "36", "Unknown",
+        "37", "Unknown",
+        "38", "Unknown",
+        "39", "Timeout",
+        "40", "Invalid TCP",
+        "98", "Unknown",
+        "99", "Unknown"
+    ];
+    let icmptypelookup=datatable(NetworkIcmpCode: int, NetworkIcmpType: string)
+        [
+        0, "Reserved",
+        1, "Destination Unreachable",
+        2, "Packet Too Big",
+        3, "Time Exceeded",
+        4, "Parameter Problem",
+        100, "Private experimentation",
+        101, "Private experimentation",
+        127, "Reserved for expansion of ICMPv6 error messages",
+        128, "Echo Request",
+        129, "Echo Reply",
+        130, "Multicast Listener Query",
+        131, "Multicast Listener Report",
+        132, "Multicast Listener Done",
+        133, "Router Solicitation",
+        134, "Router Advertisement",
+        135, "Neighbor Solicitation",
+        136, "Neighbor Advertisement",
+        137, "Redirect Message",
+        138, "Router Renumbering",
+        139, "ICMP Node Information Query",
+        140, "ICMP Node Information Response",
+        141, "Inverse Neighbor Discovery Solicitation Message",
+        142, "Inverse Neighbor Discovery Advertisement Message",
+        143, "Version 2 Multicast Listener Report",
+        144, "Home Agent Address Discovery Request Message",
+        145, "Home Agent Address Discovery Reply Message",
+        146, "Mobile Prefix Solicitation",
+        147, "Mobile Prefix Advertisement",
+        148, "Certification Path Solicitation Message",
+        149, "Certification Path Advertisement Message",
+        150, "ICMP messages utilized by experimental mobility protocols such as Seamoby",
+        151, "Multicast Router Advertisement",
+        152, "Multicast Router Solicitation",
+        153, "Multicast Router Termination",
+        154, "FMIPv6 Messages",
+        155, "RPL Control Message",
+        156, "ILNPv6 Locator Update Message",
+        157, "Duplicate Address Request",
+        158, "Duplicate Address Confirmation",
+        159, "MPL Control Message",
+        160, "Extended Echo Request",
+        161, "Extended Echo Reply",
+        200, "Private experimentation",
+        201, "Private experimentation",
+        255, "Reserved for expansion of ICMPv6 informational messages"
+    ];
+    let protocollookup=datatable(
+        protocol: string,
+        NetworkProtocol: string,
+        NetworkProtocolVersion: string
+    )[
+        "tcp", "TCP", "",
+        "tcp/ip", "TCP", "",
+        "udp", "UDP", "",
+        "udp/ip", "UDP", "",
+        "icmp", "ICMP", "IPV4",
+        "icmp6", "ICMP", "IPV6",
+        "0", "HOPOPT", "",
```

Comment:
For number based protocol, please use the helper function _ASIM_LookupNetworkProtocol

Reply:
Done